app/controllers/transactions_controller.rb

```diff
@@ -100,10 +100,28 @@ def create
   end
 
   def show
-    transaction_conversation = MarketplaceService::Transaction::Query.transaction_with_conversation(
-      params[:id],
-      @current_user.id,
-      @current_community.id)
+    is_admin = @current_user.has_admin_rights_in?(@current_community)
+    role = :participant
+
+    participant_transaction_conversation = MarketplaceService::Transaction::Query.transaction_with_conversation(
+      transaction_id: params[:id],
+      person_id: @current_user.id,
+      community_id: @current_community.id)
+
+    admin_transaction_conversation =
+      if is_admin && feature_enabled?(:admin_conversations) && participant_transaction_conversation.nil?
+        MarketplaceService::Transaction::Query.transaction_with_conversation(
+          transaction_id: params[:id],
+          community_id: @current_community.id)
+      else
+        nil
+      end
```

Comment:
Could this be written more succinctly by chaining Maybe when the if has participant_transaction_conversation.nil? check anyway?